warehouse/ingest-core/src/main/java/datawave/ingest/table/config/ShardTableConfigHelper.java

```diff
@@ -203,6 +205,16 @@ protected void configureGidxTable(TableOperations tops) throws AccumuloException
                 aggClass = KeepCountOnlyUidAggregator.class.getName();
             }
             
+            if (conf.get(MAX_INDEX_UIDS) != null) {
+                String maxUids = conf.get(MAX_INDEX_UIDS);
+                String maxUidProperty = stem + PropogatingIterator.getOptString("*", GlobalIndexUidAggregator.MAX_UIDS);
```

Comment:
Just want to make sure I understand this correctly.  Is the idea that `*` means that this setting applies to all column families?  Does the logic in PropogatingIterator bear that out?  It doesn't look like it to me.  Looks like it looks for an exact column family match.